change/@microsoft-fast-foundation-ec51332e-f013-4ae6-9471-cc619f17159e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "chore: call change when clear button is clicked",
```

Comment:
Isn't "chore" for something that doesn't affect released packages?  Seems like a "feat" or "fix" to me.

Reply:
```suggestion
  "comment": "call change when search component clear button is clicked",
```

Reply:
I think we can drop that and make the actual change file a bit more clear